app/src/main/java/com/example/androidtrainingproject/login/LoginActivity.kt

```diff
@@ -88,20 +87,26 @@ fun LoginScreen(navigateToProductDetails: () -> Unit = {}) {
             },
         )
 
-        if (showWrongPasswordMessage.value) {
-            Text("Invalid login credentials", color = Color.Red)
+        loginViewModel.isLoginSuccessful?.let { isLoggedIn ->
+            if (isLoggedIn) {
+                navigateToProductDetails()
+                Toast.makeText(LocalContext.current, "Login successful", Toast.LENGTH_SHORT).show()
```

Comment:
Maybe not really needed if it is a successful login, after all the user is navigated to a different screen.

Reply:
i guess it makes sense, I've removed it